source/frontend/src/features/documents/documentDetail/DocumentDetailContainer.tsx

```diff
@@ -95,9 +95,8 @@ export const DocumentDetailContainer: React.FunctionComponent<
           const mayanMetadataResult = metadataResponse.payload?.results;
 
           let mayanFileId: number | undefined = undefined;
-          if (exists(mayanMetadataResult) && mayanMetadataResult.length > 0) {
-            const document = mayanMetadataResult[0].document;
-            mayanFileId = document?.file_latest?.id;
+          if (exists(detailResponse)) {
```

Comment:
This meant that if a document had not metadata, the mayanFileId would never be fetched, and the file_latest api would be used instead. This is likely why this bypassed testing.